### PR TITLE
add russian translate

### PR DESCRIPTION
**Pull request by Lukentui**
with Russia from love :3

### CODE REVIEW

**Comment by oliverschwendener** [comment]
Thanks for your contribution! 🙌 

There are 2 things missing:
- [ ] Please add Russian to `language.ts`
- [ ] Please add the russian translation set to the switch statement in `translation-set-manager.ts`

**Comment by Lukentui** [comment]
Added)

**Comment by oliverschwendener** [comment]
Can you please make just one PR?

**Comment by Lukentui** [comment]
I'll try

**Comment by Lukentui** [comment]
that's right?

File: src/common/translation/language.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export enum Language {
     English = "English",
     Deutsch = "Deutsch",
+    Russian = "Russian",
```
Comment (oliverschwendener) [inline src/common/translation/language.ts]: @Lukentui Can you add the russian word for "Russian" between the double quotes?

**Comment by Lukentui** [comment]
like this?

**Pull request merged**